cluster-autoscaler/cloudprovider/alicloud/alicloud_auto_scaling_group.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
2016 ?

Reply:
I'll change the Copyright to the latest version.